### ISSUE TITLE
analog stick coordinate reporting

### ISSUE THREAD

**Issue by Batcastle** [zeth/inputs#74]
So this entire issue is just trying to figure out how the coordinates for the analog sticks are reported. More specificity, what is the coordinate system? For instance, is the X- and Y-axis like this:
```
Y
|
|
|                 * - default orientation
|    origin
|  / 
------------------------------------- X
```

or are they like this:
```
                Y
                |   origin AND default orientation
                | / 
----------------*-------------------  X
                |
                |
```
also, knowing how far the X and Y axis go would be useful too. That way, a dev can convert from one of the above graphs to another with just some simple subtraction to both X and Y coordinates.

**So, in a nutshell:**
 1. What quadrants of the coordinate plane does this work with? Just one? Or all 4?
 2. How far do both axis go?

**Comment by Avril112113** [comment]
1) For my XBox controller it dose go into the negatives, so origin and default are 0, 0
2) Im trying to figure this one out my self, I've seen for my XBox controller that it ranges between -32768 to 32767
With both of what I said, I think this may vary for each type of controller?
